load-tests/simple_producer_consumer/scripts/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+# Copyright 2022 WSO2 Inc. (http://wso2.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ----------------------------------------------------------------------------
+# Execution script for ballerina performance tests
+# ----------------------------------------------------------------------------
+set -e
+source base-scenario.sh
+
+echo "----------Running Load Test----------"
+java -jar $scriptsDir/load_test.jar "Kafka Simple Producer Consumer" "$resultsDir/summary.csv"
```

Comment:
```suggestion
java -jar $scriptsDir/load_test.jar "Nats Simple Producer Consumer" "$resultsDir/summary.csv"
```

Reply:
And also we need to compile the load_test and commit the jar to the scripts directory